api/api_hierarchical.go

```diff
@@ -13,28 +13,29 @@ import (
+	CrossMsgResolve(ctx context.Context, id address.SubnetID, c cid.Cid, from address.SubnetID) ([]types.Message, error)                                                                                                    // perm:read
+	LockState(ctx context.Context, wallet address.Address, actor address.Address, subnet address.SubnetID, method abi.MethodNum) (cid.Cid, error)                                                                           // perm:write
+	UnlockState(ctx context.Context, wallet address.Address, actor address.Address, subnet address.SubnetID, method abi.MethodNum) error                                                                                    // perm:write
+	InitAtomicExec(ctx context.Context, wallet address.Address, inputs map[string]sca.LockedState, msgs []types.Message) (cid.Cid, error)                                                                                   // perm:write
+	ListAtomicExecs(ctx context.Context, id address.SubnetID, addr address.Address) ([]sca.AtomicExec, error)                                                                                                               // perm:read
+	ComputeAndSubmitExec(ctx context.Context, wallet address.Address, id address.SubnetID, execID cid.Cid) (sca.ExecStatus, error)                                                                                          // perm:write
+	AbortAtomicExec(ctx context.Context, wallet address.Address, id address.SubnetID, execID cid.Cid) (sca.ExecStatus, error)                                                                                               // perm:write
+	SubnetChainNotify(context.Context, address.SubnetID) (<-chan []*HeadChange, error)                                                                                                                                      // perm:read
+	SubnetChainHead(context.Context, address.SubnetID) (*types.TipSet, error)                                                                                                                                               // perm:read
+	SubnetStateGetActor(ctx context.Context, id address.SubnetID, addr address.Address, tsk types.TipSetKey) (*types.Actor, error)                                                                                          // perm:read
+	SubnetStateWaitMsg(ctx context.Context, id address.SubnetID, cid cid.Cid, confidence uint64, limit abi.ChainEpoch, allowReplaced bool) (*MsgLookup, error)                                                              // perm:read
+	SubnetGetActorStateValidators(ctx context.Context, id address.SubnetID) (string, error)                                                                                                                                 // perm:read
```

Comment:
What about calling it `SubnetGetValidators`?

Reply:
fixed